lambdas/src/utils/SmartContentClient.ts

```diff
@@ -99,6 +102,18 @@ export class SmartContentClient implements ContentAPI {
     throw new Error('New deployments are currently not supported')
   }
 
+  buildEntity({ type, pointers, files, metadata }: BuildEntityOptions): Promise<DeploymentPreparationData> {
```

Comment:
I'm not sure what those no implemented methods are

Reply:
It's this 
https://github.com/decentraland/catalyst-client/pull/90